src/sphinx_ansible_theme/theme.conf

```diff
@@ -18,6 +18,7 @@ prev_next_buttons_location = bottom
 style_external_links = False
 style_nav_header_background =
 vcs_pageview_mode =
+documentation_home_url =
```

Comment:
```suggestion
documentation_home_url = /
```

Reply:
Also, move this to the bottom, separately from the vars inherented from the parent theme.

Reply:
Implemented in 79ebde21fe6279a39f489c954865033f85a00089 and 2957a18f6687b4a859ae58ca2388f45b5a3f1663.